aws/service.ts

```diff
@@ -177,6 +177,26 @@ function getOrCreateRepository(imageName: string): aws.ecr.Repository {
     if (!repository) {
         repository = new aws.ecr.Repository(imageName.toLowerCase());
         repositories.set(imageName, repository);
+
+        // Set a default lifecycle policy s.t. only the 10 most recent images are retained.
```

Comment:
10 => 1

Reply:
(and rework sentence)

Reply:
Also perhaps expand on this comment to highlight that we always tag only `latest`, so we keeping at most one untagged image will ensure that we clean up old images that are no longer being used to avoid unbounded growth of the ECR repository.